### DISCUSSION TITLE
Local Client MySQL Router 8.0

### DISCUSSION THREAD

**Discussion by adibokay** [dbeaver/dbeaver#20094] [Q&A]
How do I get MySQL Router 8.0 under connection settings > Advanced? The only options I get are MySQL Binaries & MySQL Binaries 5.

I have tried using DBeaver version 22.3.5 but still unable to solve the problem. Please help thank you

**Comment by flambchap** [comment]
Hello @adibokay. For certain operations (such as backup and restore) DBeaver uses `mysqldump` and related binaries. Since MySQL Router 8 doesn't come with these binaries, there is no point in providing the path to it in the connection settings.
